package.json

```diff
@@ -43,37 +43,37 @@
   },
   "devDependencies": {
     "@cdk8s/projen-common": "0.0.530",
-    "@cdktf/provider-aws": "^17.0.11",
+    "@cdktf/provider-aws": "^19.0.0",
     "@types/fs-extra": "^11.0.4",
-    "@types/jest": "^27",
+    "@types/jest": "^29",
     "@types/node": "16.18.78",
     "@typescript-eslint/eslint-plugin": "^7",
     "@typescript-eslint/parser": "^7",
     "cdk8s": "^2.68.4",
     "cdk8s-cli": "^2.198.154",
-    "cdktf": "^0.19.1",
+    "cdktf": "^0.20.7",
     "cdktf-cli": "^0.20.7",
     "constructs": "^10.3.0",
     "eslint": "^8",
     "eslint-import-resolver-typescript": "^3.6.1",
     "eslint-plugin-import": "^2.29.1",
     "fs-extra": "^11.2.0",
-    "jest": "^27",
+    "jest": "^29",
     "jest-junit": "^15",
     "jsii": "^5",
     "jsii-diff": "^1.100.0",
-    "jsii-docgen": "^9.2.2",
+    "jsii-docgen": "^10.4.16",
     "jsii-pacmak": "^1.100.0",
     "jsii-rosetta": "^5",
     "projen": "^0.82.8",
     "standard-version": "^9",
-    "ts-jest": "^27",
+    "ts-jest": "^29",
     "ts-node": "^10.9.2",
     "typescript": "^5.5.2"
   },
   "peerDependencies": {
     "cdk8s": "^2.68.11",
-    "cdktf": "^0.19.1",
```

Comment:
This shouldn't be required because the `^` means any minor version above 19 can be used by peers.

Reply:
@iliapolo I don't believe that is how it works. When a package version has a zero as the major version the caret will not increment the minor version, only the patch.

See this for an example:

https://semver.npmjs.com

Use cdktf and specify ^0.19.0 as the version. You'll see 0.20 package versions are not listed.

Reply:
@BrentSouza is correct. The notion of patch versions only getting bumped on dependencies below 1.0.0 is npm update docs: https://docs.npmjs.com/cli/v10/commands/npm-update#caret-dependencies-below-100